extension/src/experiments/model/collect.ts

```diff
@@ -228,7 +229,7 @@ const collectHasRunningExperiment = (
   acc: ExperimentsAccumulator,
   experiment: Experiment
 ) => {
-  if (experiment.running) {
+  if (experiment.status === ExperimentStatus.RUNNING) {
```

Comment:
Instead of checking for the status everywhere, should we add getters like `isRunning()`, `isQueued()`...